modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1415,7 +1415,7 @@ GrainTracker::updateFieldInfo()
         entity_value = _vars[curr_var]->getNodalValue(node);
       }
 
-      if (tmp_map.find(entity) == tmp_map.end() || entity_value > tmp_map[entity])
+      if (entity_value != std::numeric_limits<Real>::lowest() && (tmp_map.find(entity) == tmp_map.end() || entity_value > tmp_map[entity]))
```

Comment:
Huh? Up in the IC you assign `-1.0` here you compare against `std::numeric_limits<Real>::lowest()`. I guess I'm lost.